scripts/prerequisites.sh

```diff
@@ -0,0 +1,6 @@
+cd ..
+# clone mx-chain-go
```

Comment:
1. Delete this comm maybe?
2. Can we change maye the script name to specify these are the prerequisites for sovereign chain, not for normal tests?

Reply:
renamed the filel